packages/yaml-editor-playground/src/App.css

```diff
@@ -34,3 +34,7 @@ body,
   --indent-color: transparent !important;
   --indent-color-active: transparent !important;
 }
+
+.highlightYaml {
```

Comment:
Do we have more css other than this file? We should move to tailwind here so that we can use other tailwind utility classes.

Reply:
This is inside yaml-editor-playground package. yaml-editor does not have any css. In the playground we can use tailwind class.